tests/playwright/src/model/core/states.ts

```diff
@@ -42,3 +42,9 @@ export enum PodState {
   Paused = 'PAUSED',
   Unknown = 'UNKNOWN',
 }
+
+export enum ArchitectureType {
```

Comment:
I would prefer to have this enum in other file. `states.ts` was intended for states of objects. I am thinking about `platforms` or something else perhaps?

Reply:
sure